README.md

```diff
@@ -284,6 +284,53 @@ output = pipeline(
 output_images = pipeline.numpy_to_pil(np.asarray(output.reshape((num_samples,) + output.shape[-3:])))
 ```
 
+Diffusers also has a Text-guided inpainting pipeline with Flax/Jax
+
+```python
+import jax
+import numpy as np
+from flax.jax_utils import replicate
+from flax.training.common_utils import shard
+import PIL
+import requests
+from io import BytesIO
+
+
+from diffusers import FlaxStableDiffusionInpaintPipeline
+
+def download_image(url):
+    response = requests.get(url)
+    return PIL.Image.open(BytesIO(response.content)).convert("RGB")
+img_url = "https://raw.githubusercontent.com/CompVis/latent-diffusion/main/data/inpainting_examples/overture-creations-5sI6fQgYIuo.png"
+mask_url = "https://raw.githubusercontent.com/CompVis/latent-diffusion/main/data/inpainting_examples/overture-creations-5sI6fQgYIuo_mask.png"
+
+init_image = download_image(img_url).resize((512, 512))
+mask_image = download_image(mask_url).resize((512, 512))
+
+pipeline, params = FlaxStableDiffusionInpaintPipeline.from_pretrained("xvjiarui/stable-diffusion-2-inpainting")
```

Comment:
Maybe we should also copy the weights to a `flax` branch in `stabilityai/stable-diffusion-2-inpainting`. What do you think @patrickvonplaten @patil-suraj?

Reply:
Given that we will soon deprecate the "branch" behavior, we should probably instead add it to "main", but before doing this we need to be sure that adding flax weights doesn't change the behavior of downloading PyTorch weights. E.g. we should add some tests that if flax weights are present still only PyTorch is downloaded. 

For me ok to merge as is for now.

Reply:
In that case I think it makes sense to merge as is, and then we can migrate the weights when the new system is in place.